tests/unittest/_torch/visual_gen/test_cosmos3_action.py

```
@@ -0,0 +1,731 @@
+        kwargs.setdefault("action_chunk_size", self.CHUNK)
+        kwargs.setdefault("action_dim", self.ACTION_DIM)
+        return prepare_action_latents(
+            mode=mode,
+            generator=torch.Generator(device="cpu").manual_seed(0),
+            device=torch.device("cpu"),
+            dtype=torch.float32,
+            **kwargs,
+        )
+
+    def test_forward_dynamics_pads_a_short_trajectory_by_holding_the_last_step(self):
+        latents, mask, clean, raw_dim = self._prepare(
```

Ruff (0.16.1)

[warning] Unpacked variable `mask` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)